xml/dhcp_vendor_class.xml

```diff
@@ -944,4 +944,14 @@
     <param pos="0" name="os.product" value="{hw.product} Firmware"/>
   </fingerprint>
 
+  <fingerprint pattern="^JNPR\.IVE$">
+    <description>Juniper SSL VPN</description>
+    <example>JNPR.IVE</example>
+    <param pos="0" name="hw.device" value="SSL-VPN"/>
+    <param pos="0" name="hw.vendor" value="Juniper"/>
+    <param pos="0" name="os.vendor" value="Juniper"/>
+    <param pos="0" name="os.product" value="ive_os"/>
```

Comment:
The correct form of the `os.product` would be "IVE OS" rather than the NVD CPE dictionary format.
[IVE OS (SSL VPN) Dates & Milestones](https://support.juniper.net/support/eol/software/ive/)

Reply:
Done.